docs/openapi/api-external-b2b-webhook-v1.yaml

```diff
@@ -61,13 +61,11 @@ info:
     ``
     </br></br>
     
-    Gli stati della notifica sono i seguenti: 
+    Gli stati della notifica che possono essere inseriti come __filterValues__ sono i seguenti: 
       <details>
         <summary>Stati</summary>
         <ul>
-          <li><strong>IN_VALIDATION </strong><em>notifica depositata in attesa di validazione.</em></li>
           <li><strong>ACCEPTED </strong><em>L'ente ha depositato la notifica con successo.</em></li>
-          <li><strong>REFUSED </strong><em>Notifica rifiutata a seguito della validazione.</em></li>
```

Comment:
Scusa, ma anche refused è stata eliminata????

Reply:
i valori che ho preso sono quelli presenti in schemas-pn-status nella definizione di: "SearchableNotificationStatus" che contiene: ACCEPTED, DELIVERING, DELIVERED, VIEWED, EFFECTIVE_DATE, PAID, UNREACHABLE, CANCELLED e tra questi non è presente REFUSED

Reply:
SearchableNotificationStatus penso siano relative alla possibilità di filtro in fase di ricerca della notifica, non per gli streams

Reply:
ho ripristinato la lista degli status filtrabili, escludendo solo IN_VALIDATION